index.js

```diff
@@ -231,6 +232,8 @@ module.exports = {
     'prefer-template': 2,
     'require-yield': 0,
     'keyword-spacing': 2,
+    'react/display-name': 0,
+    'react/jsx-curly-spacing': [2, 'always'],
```

Comment:
👍 

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md

Reply:
This would be the only one I disagree with, as the airbnb consensus actually dictates the opposite: https://github.com/airbnb/javascript/blob/master/react/README.md